src/annotate.ts

```diff
@@ -24,10 +13,10 @@ export default async function annotate(field: string, term: string) {
 
   const ontology = ontologyDict[field];
 
-  const excludedTerms = ['n/a', 'na', 'none', 'not available', 'other', 'unavailable', 'unknown', 'unspecified'];
+  const excludedTerms = ['n/a', 'na', 'none', 'not available', 'not specified', 'other', 'unavailable', 'unknown', 'unspecified'];
```

Comment:
I wouldn't do this filtering here, but is arguable, so yeah. I won't complain if there is a good logic for it.

Reply:
Ideally we won't have those terms at all. I would like to filter them out at the mapper stage, but having a second layer here just helps to catch any that slip through.